src/stories/_SemanticSystem.stories.tsx

```diff
@@ -1,14 +1,29 @@
-import {
-  Div, Flex, Text, useTheme,
-} from 'honorable'
+import styled, { useTheme } from 'styled-components'
 
 import Divider from '../components/Divider'
 
+import { mixins } from '../mixins'
+
 export default {
   title: 'Semantic System',
   component: null,
 }
 
+const ItemLabel = styled.div`
+  ${mixins.text.caption}
+  margin-top: var(--space-xxsmall);
```

Comment:
IMHO it would be more convenient to use `theme` object instead of CSS variables. IDE would also be able to help with suggestions and autocomplete.

#### Example
I have been playing with it a bit to see what would be convenient to use.
[config](https://github.com/floreks/ovigos.io/blob/master/src/theming/theme.ts) | [usage](https://github.com/floreks/ovigos.io/blob/master/src/components/Layout/component.tsx)

Reply:
Oh, I see using style object syntax makes accessing the theme values far less cumbersome. I think in writing regular css is easier in general (especially when dealing with media queries), but I think the advantages of JS style objects might outweigh the downsides for our use case. Do you think the partials should go in the theme instead of pulled in from the `mixins` export? Any thoughts on how to structure that? `theme.partials`?

Reply:
I guess it should also be possible to write media queries using the object syntax. It still allows inline CSS usage. In example:
```typescript
const SectionTitle = styled.div({
  fontSize: '25px',
  display: 'flex',
  '@media(min-width: 788px)': {
    fontSize: '40px'
  }
})
```

Personally, I don't have any strong opinions on how to structure partials. I would just try to avoid using different approaches. Keeping partials in the theme object sounds like a good idea if we will go with a style object syntax. We should be able to greatly benefit from the IDE support.

Reply:
Okay, I think style objects are probably the way to go, and will probably make migrating older honorable-based components simpler. Just updated the PR to use style object syntax and include partials in the theme. PTAL.